src/point_one/fusion_engine/messages/sta5635.h

```diff
@@ -83,6 +83,28 @@ struct P1_ALIGNAS(4) STA5635CommandResponse : public MessagePayload {
   uint8_t data[4] = {0};
 };
 
+/**
+ * @brief IQ sample data from an STA5635.
+ *        (@ref MessageType::STA5635_IQ_DATA, version 1.0).
+ * @ingroup sta5635
+ */
+struct P1_ALIGNAS(4) STA5635IQData : public MessagePayload {
+  static constexpr MessageType MESSAGE_TYPE = MessageType::STA5635_IQ_DATA;
+  static constexpr uint8_t MESSAGE_VERSION = 0;
+
+  uint8_t reserved[4] = {0};
+
+  /**
+   * @note
+   * This message ~version~ struct contains no data fields.
```

Comment:
This first sentence and the `<br>` isn't needed anymore (and the ~version~ was supposed to be omitted).

Also, this note still needs to move and be part of the class level doxy instead. You can't have a floating doxygen block that isn't attached to a class or a member variable, doxygen will just drop it in that case.